lib/infra_conversion_throttler.rb

```diff
@@ -1,24 +1,42 @@
 class InfraConversionThrottler
   def self.start_conversions
+    $log&.debug("InfraConversionThrottler.start_conversions")
```

Comment:
I know the previous code had this...but why do you need the `&.` for `$log`?  This is a global var that is set up by vmdb_loggers and will never be nil

Reply:
Included `Vmdb::Logging` and updated to use `_log`.